porogo/frontend/static/styles/views/_home.scss

```diff
@@ -66,6 +66,7 @@
 
     h2 {
         -webkit-text-stroke: 1px;
```

Comment:
Is this still needed with new font size?

Reply:
Didn't try without it. Let me know if it changes anything at all.